controllers/cloud.redhat.com/providers/kafka/strimzi.go

```diff
@@ -496,9 +500,62 @@ func NewStrimzi(p *providers.Provider) (providers.ClowderProvider, error) {
 		},
 	}
 
+	createNetworkPolicies(p)
+
 	return kafkaProvider, kafkaProvider.configureBrokers()
 }
 
+func createNetworkPolicies(p *providers.Provider) error {
+	appList, err := p.Env.GetAppsInEnv(p.Ctx, p.Client)
+	if err != nil {
+		return err
+	}
+
+	namespaceSet := map[string]bool{}
+
+	np := &networking.NetworkPolicy{}
+	nn := types.NamespacedName{
+		Name:      p.Env.Name,
```

Comment:
should we make it the same name as the kafka instance?  i.e. `getKafkaName(p.Env)`?